public/css/module.less

```diff
@@ -1,5 +1,11 @@
 // Icinga Web 2 X.509 Module | (c) 2018 Icinga GmbH | GPLv2
 
+@segment-color-0: #FF595E;
+@segment-color-1: #FFCA3A;
+@segment-color-2: #8AC926;
+@segment-color-3: #1982C4;
+@segment-color-4: #6A4C93;
```

Comment:
I guess your intention by putting this into `.cert-donut`, was that it emphasizes their meaning? Nothing speaks against a more specific prefix then, e.g. `@cert-donut-segment-0`.